### ISSUE TITLE
List of methods to be implemented

### ISSUE THREAD

**Issue by coderhs** [werner/crystal_on_steroids#3]
Listing out the methods that is pending to be implemented. Methods information taken from `ActiveSupport` - core_ext

- [x] Array#in_groups
- [x] Array#from
- [x] Array#to
- [x] Array#without
- [ ] Array#wrap
- [ ] Array#

**Comment by werner** [comment]
Thanks!, I'll include them eventually.

**Comment by coderhs** [comment]
@werner I would like to help out with them as well, I am going to list them all here so that we can keep track of it.

**Comment by coderhs** [comment]
Array#slice and Array#take they are not from activesupport but I feel it would be good to have.

**Comment by werner** [comment]
Yes!, I agree, I want this shard not to be an Active Support exact clone, instead a group of methods that can be useful to anyone, recently I discovered a gem that functions in the way I want: https://github.com/bbatsov/powerpack
